.github/workflows/auto-update.yml

```
@@ -3,36 +3,41 @@ name: Auto update
 env:
   BUILDER_TOOLS_URL: https://github.com/flatpak/flatpak-builder-tools.git
   BUILDER_TOOLS_BRANCH: master
+  TOOLS_DIR: builder-tools
 
 on:
-  schedule:
-    - cron: '45 6 * * *'
+  pull_request: {}
   workflow_dispatch: {}
 
 jobs:
   update:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
       - name: Install dependencies
         run: |
           set -e
-          sudo apt install python3-toml python3-aiohttp
+          sudo apt install \
+            python3-aiohttp \
+            python3-toml
 
       - uses: actions/checkout@v2
 
-      - name: Pull before
-        run: git pull
-
       - name: Download flatpak-builder-tools
         run: |
           git clone -b ${BUILDER_TOOLS_BRANCH} \
                        ${BUILDER_TOOLS_URL} \
-                       builder-tools
+                       ${TOOLS_DIR}
 
       - name: Run update
+        env:
+          GIT_AUTHOR_NAME: Flatpak External Data Checker
+          GIT_COMMITTER_NAME: Flatpak External Data Checker
+          GIT_AUTHOR_EMAIL: ${{ github.actor }}@users.noreply.github.com
+          GIT_COMMITTER_EMAIL: ${{ github.actor }}@users.noreply.github.com
+          EMAIL: ${{ github.actor }}@users.noreply.github.com
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GIT_LFS_SKIP_SMUDGE: '1'
         run: |
           set -e
-          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
-          git config user.name "Workflow trigger"
-          ./update.py veloren-git.json -g builder-tools/cargo/flatpak-cargo-generator.py
+          ./update-generated-sources.sh
           git push https://${GITHUB_ACTOR}:${{ secrets.GITHUB_TOKEN }}@github.com/${GITHUB_REPOSITORY}
```

I believe we shouldn't need to do push over http
```suggestion
          git push
```